docs/wiki/configurator/firmware-flasher-tab.md

```diff
@@ -150,27 +150,27 @@ For CRSF, ELRS, FPORT or GHST protocols, this is included by default to simplify
-| PIN IO                    | Enable PINIO                                                                                                  |
-| RACE PRO                  | see [Betaflight 4.5 Release notes](/docs/wiki/release/betaflight-4-5-release-notes#161-race-pro-build-option) |
-| Servos                    | Enable Servo support                                                                                          |
-| VTX                       | Enable VTX                                                                                                    |
+| Option                    | Description                                                                                                                                                                            |
+| :------------------------ | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| AKK (SA Fix)              | SmartAudio patch for AKK hardware                                                                                                                                                      |
+| Acro Trainer              | Enable Acro Trainer support                                                                                                                                                            |
+| Batt. Continue            | See [#11084](https://github.com/betaflight/betaflight/pull/11084)                                                                                                                      |
+| Cam. Control              | Enable Camera Control                                                                                                                                                                  |
+| Dashboard                 | Enable external i2c Oled Display device (to be deprecated)                                                                                                                             |
+| EMFAT (AutoRun, Icon)     | Enable FAT emulation and icon for onboard flash or MSC                                                                                                                                 |
+| ESC Serial (SK) Inc. 4way | Enable SimonK and ESC Serial support for flashing via 4way interface                                                                                                                   |
+| Flash Storage             | 4.4 and earlier only. Enables Blackbox Flash Storage support. In 4.5, is auto-included via the FC config file. To manually add Flash to a 4.5 build, enter `FLASH` in `Custom Defines` |
+| FrSky OSD                 | 4.4 and earlier only. For 4.5, enter `FRSKYOSD` in `Custom Defines`                                                                                                                    |
```

Comment:
FrSky OSD is available in Other Options.

Reply:
Thank you - for some odd reason I thought it wasn't.  Fixed.